### DISCUSSION TITLE
Économise du chauffage grâce à la domotique ça marche ! - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#57] [Comments]
# Économise du chauffage grâce à la domotique ça marche ! - Haade.fr

Dans le cadre d’un nouveau concept vidéo présenté par August sur youtube, je vais te détailler l’installation d’une chaudière au gaz domotisée.

https://haade.fr/fr/blog/economise-du-chauffage-grace-a-la-domotique-ca-marche

**Comment by mdlneo57** [comment]
Bonjour,
Votre tuto m'intéresse fortement mais j'ai du mal à vous suivre avec ces mini vidéos (GIF) qui sont trop rapide, c'est bien dommage.
J'ai commencé à suivre pas à pas votre méthode mais arrivé à la partie de l'automatisation du mode confort avec boost vous m'avez perdu. Ne serait-il pas possible de proposer une vidéo (ou on pourrait faire pause) avec une meilleure qualité de résolution.

**Reply by haade-administrator** [reply]
Bonjour normalement tout est dans l'article du blog, les gifs sont un peu rapide mais compréhensible il suffit de se repérer dans ha, l'article fait déjà 14 minutes de lecture, de faire des gifs animés me fait gagner en longueur de texte.

**Comment by SpiteFire95** [comment]
Bonjour,

est-ce que en utilisant les mêmes composants que vous (voir en ajoutant des tête en plus et des capteurs, suivant mes besoins) c'est possible sur HA d'utiliser Versatile Thermostat? ou faut-il suivre votre procédure pour que cela fonctionne ?
merci pour votre retour et bonne année :)

**Reply by haade-administrator** [reply]
Ah **oui tu peux utiliser versatile thermostat** sans soucis, je ne l'ai pas fait pour une telle installation car c'est une usine à gaz, il y a tellement de fonctions qu'on a tendance à s'y perdre.

**Comment by jmb1669** [comment]
Bonjour,

Super article mais vraiment dommage que les GIFs annimés soit si peu utilisable. J'ai du les convertir en vidéo pour les suivrent. J'ai parcourru le web et c'est un des articles les plus compréhensible sur le sujet. Je comprend que ça prend beaucoup de temps pour les GIFs, mais de courtes vidéos arraient fait l'affaire et beaucoup plus de personnes pourraient en profiter cequi pourrait augmenter le nombre de vue. Félicitation pour l'article !

**Comment by haade-administrator** [comment]
Merci pour ce retour, tu es déjà le deuxième à me faire cette remarque, je vais tâcher de faire des vidéos à l'avenir

**Comment by julVw** [comment]
Bonjour,
J'aimerais partir sur une sonoff ihost et je me demande si le paramétrage est identique ou s'il y a des différences?
Et j'ai vu qu'il y avait une version 2Gb et une 4Gb, la 2Gb suffirait pour faire juste ce genre de contrôle de chauffage? Je n'aurais rien d'autre à domotiser par la suite.
Merci d'avance.

**Reply by haade-administrator** [reply]
Il y a des différences au niveau de l'automatisation mais je n'ai testé que jusqu'à la version 1.12 du firmware: [test sonoff ihost](https://haade.fr/fr/blog/test-et-deballage-passerelle-sonoff-ihost-RV1126),  maintenant ils [sont à la v2.1.0 ](https://sonoff.tech/product-review/product-insight/ihost-version-updates-information-faq/), je te conseil de partir sur la 4gb, c'est une bonne box qui intègre bien la gamme sonoff

**Reply by julVw** [reply]
Merci pour la réponse, je vais prendre le temps de tester et tout paramètrer pour être prêt pour les beaux jours. Et au niveau des thermostat d'ambiances il y a les SONOFF SNZB-02D c'est la même choses que ceux que vous conseillez à part qu'il y a un affichage? Je trouvais ça sympa d'avoir l'information dans les pièces.

**Reply by haade-administrator** [reply]
oui mais ça consomme plus d'énergies

**Comment by CoolsPops** [comment]
Bonjour 

Combien de temps dure les piles des têtes thermostatiques ?

En enocean, les têtes n'ont pas de piles. Mais je ne vois pas ce type de produit en zigbee.

Merci

**Reply by haade-administrator** [reply]
ça dépend des marques mais elles durent plus d'une année
